docker/Dockerfile

```diff
@@ -206,18 +209,23 @@ COPY VERSION /project/avalon/
 COPY ./bin /project/avalon/bin
 COPY ./common/cpp /project/avalon/common/cpp
 
+WORKDIR /opt/intel
+
 RUN packages="libsgx-dcap-ql-dev libsgx-dcap-quote-verify libsgx-dcap-quote-verify-dev"; \
     if [ "$DISTRO" = "bionic" ] ; then \
       # Add Intel SGX repo to apt sources and install Intel SGX PSW packages
       echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | tee /etc/apt/sources.list.d/intel-sgx.list; \
       wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add - ; \
       apt-get update; \
+      /project/avalon/scripts/install_packages -c install -q "$packages"; \
     elif [ "$DISTRO" = "centos" ] ; then \
-      wget https://download.01.org/intel-sgx/sgx-linux/2.10/distro/centos8.1-server/sgx_rpm_local_repo.tgz; \
+      wget https://download.01.org/intel-sgx/sgx-dcap/1.8/linux/distro/centos8.1-server/sgx_rpm_local_repo.tgz; \
```

Comment:
Any specific reason to stick to a lower version? I see 1.9 as well in the repo.

Reply:
Might be because when DCAP feature was implemented, the latest version was 1.8